src/lib/types.ts

```diff
@@ -66,7 +66,7 @@ export interface Saga<State, Payload> {
 
 export type AnySaga = Saga<any, any>;
 
-export type ErrorHandler = (error: unknown) => void;
+export type ErrorHandler = <Payload>(error: unknown, action: Action<Payload>) => void;
```

Comment:
Does it make sense to type `error` here as unknown? It should always be an `Error` object.

Reply:
Since one can `throw` anything I don't think we could be more specific.